### PR TITLE
feat(markdown): Add support for quoted emails

### LINKED ISSUE #467: markdown: Add support for quoted text
When replying from an email, the notification content is embedded in the reply.

On github.com, it's displayed as a foldable section, folded by default:
screen shot 2017-10-11 at 12 22 21 pm

Currently the app doesn't take care of it:
screen shot 2017-10-11 at 12 22 05 pm

Should be handled in the same way.

### PR DESCRIPTION
**Pull request by machour**
Introduced a `<ToggleView>` component which will also come handy when we add PR review comments.

gif

Closes #467

### CODE REVIEW

File: src/components/github-htmlview.component.js [hunk 1]
```diff
@@ -65,6 +65,17 @@ const styles = {
   a: linkStyle,
 };
 
+const quotedEmailToggleStyle = {
+  backgroundColor: colors.greyMid,
+  paddingLeft: 4,
+  paddingRight: 4,
```
Comment (lex111) [inline src/components/github-htmlview.component.js]: If use `paddingHorizontal` instead `paddingLeft` and `paddingRight`, will it work?

Comment (machour) [reply src/components/github-htmlview.component.js]: yes, thank you for the reminder 👍

File: src/components/github-htmlview.component.js [hunk 2]
```diff
@@ -246,6 +261,33 @@ export class GithubHtmlView extends Component {
...
+                    siblings,
+                    parent,
+                    defaultRenderer
+                  )}
+                </ToggleView>
+              );
+            }
+          }
+
+          return undefined;
```
Comment (lex111) [inline src/components/github-htmlview.component.js]: I can not understand why `undefined` instead of `null`?

Comment (machour) [reply src/components/github-htmlview.component.js]: From react-native-htmlview documentation: 
> renderNode: a custom function to render HTML nodes however you see fit. If the function returns undefined (not null), the default renderer will be used for that node.

Comment (lex111) [reply src/components/github-htmlview.component.js]: I'm understood, thank you.

**Pull request merged**